core/src/main/java/com/nisovin/magicspells/spells/targeted/EntityDataApplySpell.java

```diff
@@ -0,0 +1,37 @@
+package com.nisovin.magicspells.spells.targeted;
+
+import org.bukkit.entity.LivingEntity;
+
+import com.nisovin.magicspells.util.*;
+import com.nisovin.magicspells.spells.TargetedSpell;
+import com.nisovin.magicspells.spells.TargetedEntitySpell;
+
+public class EntityDataApplySpell extends TargetedSpell implements TargetedEntitySpell {
+
+	private final EntityData entityData;
+
+	public EntityDataApplySpell(MagicConfig config, String spellName) {
+		super(config, spellName);
+
+		entityData = new EntityData(getConfigSection("entity"));
```

Comment:
This should be `entityData = new EntityData(getConfigSection("entity"), true);`, to avoid the options that have default values from always applying.